### ISSUE TITLE
error when installing bower

### ISSUE THREAD

**Issue by OzmoMac** [userexec/Pi-Kitchen-Dashboard#2]
<h2>
Hi when i try to install bower it gives me this error

i have installed node

pi@PiServer ~ $ sudo npm install -g bower

/usr/local/bin/node: /usr/lib/arm-linux-gnueabihf/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /usr/local/bin/node)
/usr/local/bin/node: /lib/arm-linux-gnueabihf/libc.so.6: version`GLIBC_2.16' not found (required by /usr/local/bin/node)

**Comment by OMJ-** [comment]
I'm also having this issue. Did you find a solution?

**Comment by userexec** [comment]
This sounds like it's missing the build-essential package. Try sudo apt-get install build-essential. Could be that's not included anymore and needs to be added to the docs.

(Testing this out now, but will need a while to get a fresh install going. Sorry I missed this question like a month ago. No idea how it slipped past me.)

**Comment by OMJ-** [comment]
I was able to get it working by following the instructions on this site.

http://blog.wia.io/installing-node-js-v4-0-0-on-a-raspberry-pi/

**Comment by userexec** [comment]
Nice find! I'll update the readme in just a bit here.

**Comment by userexec** [comment]
Ack. So much has changed since I last used this project. More updates to the readme will be necessary, especially regarding Chromium, so expect to run into further problems especially if you're using a B/B+. Working through them now and will have updated instructions as soon as I figure out how to get a clean run on this install.

**Comment by userexec** [comment]
Alright, think I got it all. The move to Raspbian Jessie and the lack of a new Chromium build along with the update to Node v4.0.0 caused some pretty far-reaching issues with the install instructions; however, they've been updated with the new commands and a switch to Midori, which can do Kiosk mode quite nicely (couldn't seem to get rid of the window border on Epiphany). I've also added a section on installing unclutter (to auto-hide the mouse pointer) and instructions for how to schedule display sleep times.
